app/models/login.rb

```diff
@@ -0,0 +1,6 @@
+class Login < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] These Devise modules should be in the User model, not Login model.

- [ ] There is no need for a Login model. Kindly delete this model.

- [ ] When a user signs up they should be required to confirm their emails before proceeding to log in. You are missing the `:confirmable` module for handling that.